.github/workflows/build-luajit.yml

```diff
@@ -5,6 +5,9 @@ on:
 
 jobs:
   build-linux:
+    strategy:
+      matrix:
+        arch: [x86_64-linux] # arm64-linux
```

Comment:
There is no runner for `arm64-linux` on github, so I didn't touch it

Reply:
There is now ;)
I've just checked in the fixes on defold/dev